docs/how-to/hip_runtime_api/initialization.rst

```diff
@@ -0,0 +1,101 @@
+.. meta::
+   :description: Initialization.
+   :keywords: AMD, ROCm, HIP, initialization
+
+********************************************************************************
+Initialization
+********************************************************************************
+
+The initialization involves setting up the environment and resources needed for
+using GPUs. The following steps are covered with the initialization:
+
+- Setting up the HIP runtime
+
+  This includes loading necessary libraries and setting up internal data
+  structures.
+
+- Querying and setting GPUs
+
+  Identifying and querying the available GPU devices on the system.
+
+- Setting up contexts
```

Comment:
I removed the finalize section, because it was a wrong title name. I just wanted to add a note, that after hipDeviceReset(), first API call re-initialize the HIP API.